admin/queue-scheduler/src/main/java/io/enmasse/queue/scheduler/QueueScheduler.java

```diff
@@ -65,7 +72,19 @@ private static String getGroupId(ProtonConnection connection) {
 
     @Override
     public void start() {
-        server = ProtonServer.create(vertx);
+        ProtonServerOptions options = new ProtonServerOptions();
+        if (certDir != null) {
+            options.setSsl(true)
+                    .setClientAuth(ClientAuth.NONE)
```

Comment:
What does ClientAuth.NONE do? We need clients to authenticate using TLS, right?

Reply:
Yes, this should be  setClientAuth(ClientAuth.REQUIRED);